src/rgw/rgw_op.cc

```diff
@@ -6317,7 +6326,12 @@ void RGWCompleteMultipart::execute(optional_yield y)
 		     << " ret=" << op_ret << dendl;
     return;
   }
+
+  jspan_context trace_ctx(false, false);
+  extract_span_context(meta_obj->get_attrs(), trace_ctx);
```

Comment:
do we need to check if this fails (not attr)?
or do we just get an empty `trace_ctx`, and, as a result, an empty `multipart_trace`?

Reply:
we get a default `trace_ctx` and then the `add_span` method will perform noop `StartSpan`. (`trace_ctx.isValid()` will return false)